docker-compose.yml

```diff
@@ -1,22 +1,22 @@
 version: "3.8"
 services:
-  # sometimes when building the controller it does not copy the contents of the datasets folder.
-  # This is probably a caching docker caching issue, which can be fixed with:
-  # 'make clean-hard'
+  sometimes when building the controller it does not copy the contents of the datasets folder.
```

Comment:
Missing "#"-Character on comments inside this whole file. Is this intended?

Reply:
no, it's not :D